File: golang/389. Find the Difference .go
func findTheDifference(s string, t string) byte {
    var total int = 0
    
    for _, c := range t {
        total += int(c)
    }
    
    for _, c := range s {
        total -= int(c)
    }
    
    return byte(total)
}

File: java/389. Find the Difference .java
class Solution {
    public char findTheDifference(String s, String t) {
        
        int total = 0;
        
        for(char c : t.toCharArray())
            total += c;
        
        for(char c : s.toCharArray())
            total -= c;
        
        return (char) total;
    }
}
